install/kubernetes/helm/istio/charts/mixer/templates/config.yaml

```diff
@@ -233,6 +237,8 @@ spec:
     httpAuthority: request.headers[":authority"] | request.host | ""
     xForwardedFor: request.headers["x-forwarded-for"] | "0.0.0.0"
     reporter: conditional((context.reporter.kind | "inbound") == "outbound", "source", "destination")
+    gRpcStatus: response.grpc_status | ""
+    gRpcMessage: response.grpc_message | ""
```

Comment:
nit: should we just make these `grpcStatus` and `grpcMessage` (no cap 'R') ?

Reply:
Thanks.. Fixed